src/templates/edit.html

```diff
@@ -142,6 +142,15 @@ <h4>{{ 'Information'|trans }}</h4>
 
 {{ include('uploads.html') }}
 
+<!-- JSON EDITOR -->
+<section class='box' id='json-editor'>
+    <i class='fas fa-list mr-1 align-baseline'></i><h3 class='d-inline'>{{ 'JSON Editor'|trans }}</h3>
+    <div id="jsoneditor"></div>
+    <div class='submitButtonDiv'>
+      <button type='submit' style='cursor:default;' disabled name='save-json-button' class='button jsonSaver'>{{ 'Saved'|trans }}</button>
```

Comment:
try to avoid inline style, also, why is this necessary? why is the name necessary?